examples/light-switch-app/asr/src/BindingHandler.cpp

```diff
@@ -280,7 +280,7 @@ void BindingHandler::PrintBindingTable()
             \t+ ClusterId %d \n \
             \t+ RemoteEndpointId %d \n \
             \t+ NodeId %d",
-                    (int) entry.fabricIndex, (int) entry.local, (int) entry.clusterId.Value(), (int) entry.remote,
+                    (int) entry.fabricIndex, (int) entry.local, (int) entry.clusterId.value_or(-1), (int) entry.remote,
```

Comment:
so previously, did this just crash if there was no value?

Reply:
Sure looks like it....

Reply:
It looks like yes ... we maybe knew this will always have a value, however I did not check. I figured I may as well replace with value_or.